<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AntennaDataToDatabaseClass</class>
 <widget class="QMainWindow" name="AntennaDataToDatabaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntennaDataToDatabase</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="but_OpenDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть папку с
 .out файлами </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_FindFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_LoadOutToDB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить .out файлы
в базу данных </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>222</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="antennadatatodatabase.qrc"/>
 </resources>
 <connections/>
</ui>
